packages/ui/src/components/wui-button/index.ts

```diff
@@ -0,0 +1,76 @@
+import type { TemplateResult } from 'lit'
```

Comment:
Button uses text and icon components, thus should be a `composite` instead. (same for it's story)